common/metrics/otel_metrics_handler_test.go

```diff
@@ -60,7 +60,7 @@ func TestMeter(t *testing.T) {
 	err := exp.Collect(ctx)
 	assert.Nil(t, err)
 
-	lib := metrictest.Library{InstrumentationName: "test"}
+	lib := metrictest.Scope{InstrumentationName: "test"}
```

Comment:
metrictest.Library was deprecated upstream - this is just keeping up with the name changes.